### DISCUSSION TITLE
Ohne NRF24L01+, nur mit Olimex ESP32-POE-ISO oder anderem PoE-Board

### DISCUSSION THREAD

**Discussion by HalliHalloSchatz** [tbnobody/OpenDTU#678] [Ideas]
Ist es möglich, nur mit dem ESP32-POE Board eine Verbindung zum Wechselrichter auf zu bauen?

Das Board wird ja bereits mit Ethernet verbunden und hat die freie WLAN Verbindung. Kann diese genutzt werden, sodass man sich das NRF24L01+ sparen kann?

Danke

Gibt es da weitere Informationen darüber, wieso das interne WLAN nicht die NRF24L01+ ersetzen kann?
Interessiert mich gerade, was die NRF24L01+ mehr kann, was dem internen WLAN Chip fehlt.

Würde mir das gerne mal anschauen... Wieso und weshalb... Es technisch verstehen. 

_Originally posted by @HalliHalloSchatz in https://github.com/tbnobody/OpenDTU/issues/667#issuecomment-1459873902_

**Comment by HalliHalloSchatz** [comment]
Ich habe ein paar Informationen beim Projekt ahoy gefunden.

**Reply by x90cr** [reply]
Und welche sind das?
Würde mich auch interessieren.
